src/specs/models/WorkRequestBodies.v1.yaml

```diff
@@ -6,3 +6,4 @@ oneOf:
   - $ref: ./work-request-bodies/WorkRequestDifferentialExpression.v1.yaml
   - $ref: ./work-request-bodies/WorkRequestListGenes.v1.yaml
   - $ref: ./work-request-bodies/WorkRequestGeneExpression.v1.yaml
+  - $ref: ./work-request-bodies/WorkRequestMitochondrialContent.v1.yaml
```

Comment:
This ref seems to point to something that does not exist?

Reply:
I edited with Spotlight Studio and seemed forgot to press save. File added to replace `WorkRequestLoadCellMetadata`